app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+  # rubocop:enable Metrics/LineLength
+  
+  class User < ApplicationRecord
+    attr_accessor :user
+
+    validates :username, presence: true
+    validates :email, presence: true, format: { with: EMAIL_REGEX, message: 'Not Valid' }
+    validates :password, presence: true
```

Comment:
- [ ] `[OPTIONAL]` instead of writing that whole regex variable, you can use this:
```Ruby
validates :email, presence: true,  format: { with: URI::MailTo::EMAIL_REGEXP } 
```